iotdb/src/main/java/org/apache/iotdb/db/utils/MathUtils.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.db.utils;
+
+import org.apache.iotdb.tsfile.common.conf.TSFileConfig;
+
+public class MathUtils {
+
+  private MathUtils() {
+    throw new IllegalStateException("Utility class");
+  }
```

Comment:
Why add this function?

Reply:
Sonar says for Utility class (whose public methods are all `static`), we need add a private construction function to avoid some call the method like `new MathUtils.method()`

Reply:
got it